Service/TracingService.php

```diff
@@ -36,6 +36,11 @@ public function injectTracingHeadersIntoCarrier(array $carrier): array
         return $this->doInjectTracingHeadersIntoCarrier($span, $carrier);
     }
 
+    public function getActiveSpan(): ?Span
+    {
+        return $this->tracer->getActiveSpan()    
```

Comment:
The pipeline reported whitespaces at the end of the line. I think we are also missing a `;` here.
```suggestion
        return $this->tracer->getActiveSpan();
```

Reply:
Sure, I will update them and will try to write an example on how it works with Symfony Messenger